src/ripple/app/main/LoadManager.cpp

```diff
@@ -32,40 +33,56 @@
 
 namespace ripple {
 
-class LoadManagerImp
-    : public LoadManager
-    , public beast::Thread
+class LoadManagerImp : public LoadManager
 {
 public:
-    using LockType = std::mutex;
-    using ScopedLockType = std::lock_guard <LockType>;
+    //--------------------------------------------------------------------------
 
-    Application& app_;
-    beast::Journal m_journal;
-    LockType mLock;
+    Application& mApp;
+    beast::Journal mJournal;
```

Comment:
I'd suggest we update the file to the new standard of a single trailing underscore for private member variables in a FOLD commit.